docs/_pages/docs/03-understand/02-robots.md

```diff
@@ -10,4 +10,48 @@ sidebar:
+
+Remember to be concise but courteous when chatting with your robot peers!
+
+## **Why the Privacy?**
+
+Prioritizing absolute privacy for the end-user provides the highest degree of protection. User data is especially ripe for exploitation by hackers and cybercriminals; to avoid such scenarios in the first place, RoboSats does not collect any end-user data.
+
+Platforms collecting personal information present a real risk to the end-user. Data breaches in just the past decade have altogether leaked billions of users' sensitive information through a combination of hackings and poor platform security.
+
+Privacy is extremely important for RoboSats; however, your transactions on RoboSats are only as private as you make them. Users should take care to use privacy-preserving methods when interacting with RoboSats and your robot peers. Refer to [Quick Start > Access](https://learn.robosats.com/docs/access/) for additional information.
+
+## **Robot Re-Use: Not Recommended**
+
+It is strongly recommended to generate a new, random robot after each trade to enhance your privacy. Robot re-use has the potential to expose end-user information if multiple orders can be linked to a single avatar.
+
+The unique token associated with each avatar is not intended for robot re-use; rather, it is intended to act like a password for accessing ongoing orders and resolving active disputes. Please safely store this token or risk never accessing that particular robot avatar again.
+
+While we may become fond of our one-time robot identity during the short time an order is made or taken, it is better to move on to a new avatar. Think of all the great times you'll have making and taking orders with new robot identities!
+
+As stated, robot re-use is ill-advised and can ultimately become a detriment to end-user privacy.
+
+## **Robot Construction Process**
+
+RoboSats references the source code for RoboHash.org as a quick way to generate fresh avatars for a website. Your robot is "built" from a unique token, a random string of characters (ZD3I7XH...), where only those characters in their exact order can generate that exact robot avatar.
+
+A token is automatically generated for you each time the RoboSats webpage is accessed. Users may re-generate as many randomized tokens as they'd like, which is strongly encouraged after each trade. As a note, you can enter any token created by yourself instead of relying on RoboSats. Because only you know the token, it is wise to safely back up your token.
+
+Under the hood, token creation in the RoboSats homepage is the process of generating and encrypting your PGP private key with your token on the client side of the app. The end-user requests from the RoboSats node an avatar and nickname generated from your encrypted token and returns to you the corresponding robot identity. See the graphic below:
+
+![RoboSats Identity Generation Pipeine](https://learn.robosats.com/assets/images/private/usergen-pipeline.png)
```

Comment:
Misspelled "Pipeline" in the brackets. Missing an "L" and should read:

`![RoboSats Identity Generation Pipeline](https://learn.robosats.com/assets/images/private/usergen-pipeline.png)`

Reply:
Do you want to push a new commit to your branch fixing this typo and remove the "Include WIP" line? Then I will merge :)

Reply:
I _think_ I pushed a new commit correctly.. let me know if I've not though. Sorry, full noob on display